ext/store/bigquery/schema.go

```diff
@@ -19,6 +19,7 @@ const (
 	KindTable         string = "table"
 	KindView          string = "view"
 	KindExternalTable string = "external_table"
+	KindFunction      string = "function"
```

Comment:
why don't we make the name as same? like, using `routine` for all or `function` for all?

Reply:
because routine are specifically named for `BigQuery` and routine basically contains TableFunction, UDF and StoreProcedure. I think its better to named it more general for other datastore as well.